monitor/prom/progress.go

```diff
@@ -55,7 +57,12 @@ func (p *Progress) Summary(command string, summary monitor.Summary, stderr strin
 		}
 	}
 	if p.profile.PrometheusPush != "" {
-		err := p.metrics.Push(p.profile.PrometheusPush, command)
+		jobName := "${COMMAND}"
+		if p.profile.PrometheusPushJob != "" {
+			jobName = p.profile.PrometheusPushJob
+		}
+		jobName = strings.ReplaceAll(jobName, "${COMMAND}", command)
```

Comment:
It looks good.
But for maximum compatibility I think we can also use `os.Expand()` instead of `strings.ReplaceAll()`.
So we can also use $COMMAND as not everybody likes to enclose environment variable around curly braces

Reply:
Had actually started a review with exactly the same comment :) .. but forgot to commit it...